os/arch/arm/src/imxrt/mpu-reg.h

```diff
@@ -49,6 +49,7 @@
 #define MPU_REG_KERN_DATA	MPU_REG1
 #define MPU_REG_USER_CODE	MPU_REG2
 #define MPU_REG_USER_DATA	MPU_REG3
+#define MPU_REG_APP		MPU_REG4
```

Comment:
Comments (line 39~46) say 4 definitions, is this also required for memory protection? Is it mandatory?

Reply:
Fixed.